README.md

```diff
@@ -1,2 +1,59 @@
 # `jest-runner-tsd`
-A jest runner that checks typescript typings
+
+A Jest runner that tests typescript typings using [tsd](https://github.com/SamVerschueren/tsd) under the hood.
+
+## Usage
+
+### Install
+
+Install `jest`, `tsd` and `jest-runner-tsd`
+
+```bash
+yarn add --dev jest tsd jest-runner-tsd
+
+# or with NPM
+
+npm install --save-dev jest tsd jest-runner-tsd
+```
```

Comment:
Woah woah woah. `jest-runner-tsd` is a simple runner that should be used in a existing project that uses jest. So no need to install `jest` and `tsd`.

Reply:
Yeah, missed it. It's corrected now